cli/cmd/plugin/unmanaged-cluster/cluster/kind.go

```diff
@@ -329,18 +327,20 @@ type dockerInfo struct {
 	Architecture string `json:"Architecture"`
 }
 
-func validateDockerInfo(output []byte) []error {
+func validateDockerInfo(output []byte) (warnings []string, issues []error) {
 	info := dockerInfo{}
 	if err := json.Unmarshal(output, &info); err != nil {
 		// Nothing else we can check, just return this error right away
-		return []error{errors.New("unable to parse Docker information")}
+		return nil, []error{errors.New("unable to parse Docker information")}
 	}
 
-	issues := []error{}
-
 	if !strings.HasSuffix(info.Architecture, "x86_64") {
-		// Only amd64 supported right now, no arm
-		issues = append(issues, errors.New("only amd64 architecture currently supported"))
+		// Only amd64 supported right now, arm is experimental. Anything else is not supported.
+		if strings.HasSuffix(info.Architecture, "arm64") {
```

Comment:
the correct suffix is `aarch64`

Reply:
Hmm, we may need a list of acceptable variations then as `arm64` is also a valid one.

Reply:
Follow up - I was crossing the go arch values with the architecture returned by `docker info`. `aarch64` is the correct one from docker.